search-commons/src/main/java/no/unit/nva/search2/ticket/TicketParameter.java

```diff
@@ -75,7 +78,7 @@ public enum TicketParameter implements ParameterKey {
     CUSTOMER_ID_NOT(FUZZY_KEYWORD, NOT_ONE_ITEM, CUSTOMER_ID_KEYWORD),
     ID(FUZZY_KEYWORD, ONE_OR_MORE_ITEM, ID_KEYWORD),
     ID_NOT(FUZZY_KEYWORD, NOT_ONE_ITEM, ID_KEYWORD),
-    EXCLUDE_SUBUNITS(IGNORED),
+    EXCLUDE_SUBUNITS(IGNORED, ONE_OR_MORE_ITEM, ORGANIZATION_ID_KEYWORD + PIPE + ORGANIZATION_IDENTIFIER_KEYWORD),
```

Comment:
instead of being creative, I set search path for `excluded `in `excluded`, (almost) just like all other parameters